OpenMEEG/libs/OpenMEEGMaths/include/matrix.h

```diff
@@ -187,7 +187,7 @@ namespace OpenMEEG {
     }
     
     inline double Matrix::frobenius_norm() const {
-    #ifdef HAVE_LAPACK
+    #if defined(HAVE_LAPACK) && !defined(NO_DLANGE)
```

Comment:
why not use dnrm2 then?

http://www.netlib.org/lapack/explore-3.1.1-html/dnrm2.f.html